src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -40,6 +40,18 @@ export interface EslintStrategy {
 export class StaticDependencies {
 	/* eslint-disable @typescript-eslint/no-explicit-any */
 	createCLIEngine(config: Record<string,any>): CLIEngine {
+		// From https://eslint.org/docs/developer-guide/nodejs-api:
```

Comment:
This isn't part of the original bug, but I was getting a lot of false errors when manually testing these changes against real repos.